tests/vf_prefix_tests.ts

```diff
@@ -0,0 +1,178 @@
+import { Stem } from 'stem';
+import { StringNumber } from 'stringnumber';
+import { Stroke } from 'strokes';
+import { System } from 'system';
+import { TabNote } from 'tabnote';
+import { TabSlide } from 'tabslide';
+import { TabStave } from 'tabstave';
+import { TabTie } from 'tabtie';
+import { TextBracket } from 'textbracket';
+import { TextDynamics } from 'textdynamics';
+import { TextFont } from 'textfont';
+import { TextNote } from 'textnote';
+import { TickContext } from 'tickcontext';
+import { TimeSignature } from 'timesignature';
+import { TimeSigNote } from 'timesignote';
+import { Tremolo } from 'tremolo';
+import { Tuning } from 'tuning';
+import { Tuplet } from 'tuplet';
+import { Vibrato } from 'vibrato';
+import { VibratoBracket } from 'vibratobracket';
+import { Voice } from 'voice';
+
+import { equal, notEqual, propEqual, QUnit, test } from './support/qunit_api';
+
+const VFPrefixTests = {
+  Start(): void {
+    QUnit.module('VF.* API');
+    test('VF.* API', this.VF_Prefix);
+  },
+
+  VF_Prefix(): void {
+    // eslint-disable-next-line
+    const VF: any = Vex.Flow;
+    equal(Accidental, VF.Accidental);
+    equal(Annotation, VF.Annotation);
+    equal(Articulation, VF.Articulation);
+    equal(Barline, VF.Barline);
+    equal(BarNote, VF.BarNote);
+    equal(Beam, VF.Beam);
+    equal(Bend, VF.Bend);
+    equal(BoundingBox, VF.BoundingBox);
+    equal(BoundingBoxComputation, VF.BoundingBoxComputation);
+    equal(ChordSymbol, VF.ChordSymbol);
+    equal(Clef, VF.Clef);
+    equal(ClefNote, VF.ClefNote);
+    equal(Crescendo, VF.Crescendo);
+    equal(Curve, VF.Curve);
+    equal(Dot, VF.Dot);
+    equal(EasyScore, VF.EasyScore);
+    equal(Element, VF.Element);
+    equal(Factory, VF.Factory);
+    equal(Flow.RESOLUTION, VF.RESOLUTION);
+    equal(Font, VF.Font);
+    equal(Fonts, VF.Fonts);
+    equal(Formatter, VF.Formatter);
+    propEqual(new Formatter(), new VF.Formatter(), 'new Formatter()');
+    equal(Fraction, VF.Fraction);
+    equal(FretHandFinger, VF.FretHandFinger);
+    equal(GhostNote, VF.GhostNote);
+    equal(Glyph, VF.Glyph);
+    equal(GlyphNote, VF.GlyphNote);
+    equal(GraceNote, VF.GraceNote);
+    equal(GraceNoteGroup, VF.GraceNoteGroup);
+    equal(GraceTabNote, VF.GraceTabNote);
+    equal(KeyManager, VF.KeyManager);
+    equal(KeySignature, VF.KeySignature);
+    equal(KeySigNote, VF.KeySigNote);
+    equal(Modifier, VF.Modifier);
+    equal(ModifierContext, VF.ModifierContext);
+    equal(MultiMeasureRest, VF.MultiMeasureRest);
+    equal(Music, VF.Music);
+    equal(Note, VF.Note);
+    equal(NoteHead, VF.NoteHead);
+    equal(NoteSubGroup, VF.NoteSubGroup);
+    equal(Ornament, VF.Ornament);
+    equal(Parser, VF.Parser);
+    equal(PedalMarking, VF.PedalMarking);
+    equal(Registry, VF.Registry);
+    equal(Renderer, VF.Renderer);
+    equal(RepeatNote, VF.RepeatNote);
+    equal(Repetition, VF.Repetition);
+    equal(Stave, VF.Stave);
+    notEqual(Stave, VF.StaveNote);
```

Comment:
Is this intended to be `notEqual`?

Reply:
Yeah, I was just asserting that Stave and **StaveNote** are not the same. I can remove that line if it looks ugly. :-)